### PR TITLE
Fix fast moving player cards with same score

### LINKED ISSUE #283: Watch another replay not canceling players form current game
bug

### PR DESCRIPTION
**Pull request by Klimonov**
Also fixed https://github.com/anthive/website/issues/283

### CODE REVIEW

**Review by EgoriusRus — APPROVED** [review]
LGTM

File: pages/game.vue [hunk 1]
```diff
@@ -106,6 +105,7 @@ export default {
           // eslint-disable-next-line
           this.gamePlayer.on(AnthivePlayer.event.TICK, data => {
             this.players = data.bots
+            this.players.sort(({ stats: a }, { stats: b }) => (b.score === a.score ? a.art - b.art : b.score - a.score))
```
Comment (kezlya) [inline pages/game.vue]: We should trigger sort not from tick but from time. And take values of tick ;)

**Review by kezlya — CHANGES_REQUESTED** [review]
change the trigger or sort

**Review by kezlya — APPROVED** [review]
Very Good. Thank you!

**Pull request merged**